docs/papers/squad-paper.md

```diff
@@ -110,26 +168,45 @@ costly.
 Funding reciepts can only be transfered to addresses without any
 existing funding reciepts, and cannot be split sold or refunded.
 
+Let's consider framing this section in terms of these principles of DAO design:
+1. How and why would one become a voter?
+* Incedentally, as a result of people using (buying tokens from) your contributions
+2. How and why would one vote?
+* To fix some problem with the terms of service, which keeps the economic system you depend on safe
+* To censor some problematic content, for the same reason
+3. How and why would one fund the DAO?
+* In order to make more money off your game design contributions
+4. How and why would one get paid by the DAO?
+* Generally: for contributing to the public goods of the Squad ecosystem
+* Examples: lawyer editing the ToS or enforcing licenses, moderators identifying content that goes against ToS
+
 #### Terms of service
 
 A legally binding agreement between participants
 
-#### System Parameters
+#### System Parameters & Requirements
 
+Params:
 * Definition bond price curves
-* Reputation award amount and frequency
-* Reputation award distribution equation
+* Contributor Reputation award amount and frequency
+* Contributor Reputation award distribution equation
   * Probably logarithmic based on tokens sold
 * Gov DAO investor reputation equation and percentage
   * Probably simple token weight
+  (it can't be worse than what you'd get from buying a ton of your own contribution, or else investors might do that,
+  which is bad for the curation markets)
+  (quadratic might be fine here, as well: yes, you can pseudo-sybil attack, but it's still more expensive because of the gas costs)
 * Funding reciept price
 * Gov DAO Funding Discount Curve
 
+Requirements:
+* 
```

Comment:
What were you thinking for requirements?

Reply:
DAO features we need in order to build this DAO